frontend/client/components/Proposal/Comments/index.tsx

```diff
@@ -125,10 +125,17 @@ class ProposalComments extends React.Component<Props, State> {
               </Button>
             </>
           ) : (
-            <Placeholder
-              title="Your email is not verified"
-              subtitle="Please verify your email to post a comment."
-            />
+            <>
+              <h4 className="ProposalComments-verify">
+                Please verify your email to post a comment.
+              </h4>
+              <MarkdownEditor
+                ref={el => (this.editor = el)}
+                onChange={this.handleCommentChange}
```

Comment:
No biggie, but ref and onChange could be dropped since this is just for looks.

Reply:
Looks like onChange is a required prop, so I'll keep it.